services/tests/test_yaml.py

```diff
@@ -56,3 +56,15 @@ def test_when_commit_has_yaml_with_owner(self, mock_fetch_yaml):
         """
         config = yaml.final_commit_yaml(self.commit, self.org)
         assert config["codecov"]["require_ci_to_pass"] is False
+
+    @patch("services.yaml.fetch_current_yaml_from_provider_via_reference")
+    def test_when_commit_has_reserved_to_string_key(self, mock_fetch_yaml):
+        mock_fetch_yaml.return_value = """
+        codecov:
+          notify:
+            require_ci_to_pass: no
+        to_string: hello
+        """
+        config = yaml.final_commit_yaml(self.commit, self.org)
```

Comment:
In case you didn't see this, I wanted to see what `final_commit_yaml` does - there is this essay about the logic 😂 
https://github.com/codecov/shared/blob/f40874907d3ddbe6993c8016c3f09306cc14a9f8/shared/yaml/user_yaml.py#L128

I think it's not a problem because your change only deals with the "ownerYaml".

I also checked what other clients than the graphql resolver for the owner yaml in this PR may be accessing the revised owner.yaml field. Searching "owner.yaml" shows nothing in shared and a couple spots in `worker` that all point back to the same `final_commit_yaml` function above. That function returns a dictionary that doesn't appear to matter if it has the extra `to_string` on it from what I can tell. So no additional work there needed from what I can tell.